### PR TITLE
su2 on sharc: updated rst file, new install and module file

### PR DESCRIPTION
**Pull request by dmrogers75**

### CODE REVIEW

**Comment by annakrystalli** [comment]
HI @dmrogers75! In response to your review request, I've had a quick look and I'm not really in a position to comment on its accuracy. It's somewhat outside my expertise. @willfurnass might be a better candidate?

**Comment by dmrogers75** [comment]
Hello Anna
Thanks for looking. Yes, I shall ask Will to review it.
Cheers,
David

File: sharc/software/apps/su2.rst [hunk 1]
```diff
@@ -13,9 +13,10 @@ The SU2 suite is an open-source collection of C++ based software tools for perfo
 Usage
 -----
 
-SU2 5.0.0 can be activated using the module file::
+SU2 5.0.0 or 6.0.0 can be activated using the module files::
```
Comment (willfurnass) [inline sharc/software/apps/su2.rst]: Note that also loads OpenMPI?

Comment (dmrogers75) [reply sharc/software/apps/su2.rst]: Added: Note that the above module files also load GCC 6.2.0 and Open MPI 2.1.1.

**Pull request merged**